{"level":"error","message":"OCR processing error: Aadhaar number already exists","service":"aadhaar-ocr","stack":"Error: Aadhaar number already exists\n    at AadhaarRepository.create (C:\\Users\\phnaj\\aadhaar-ocr\\backend\\dist\\repositories\\aadhar.repository.js:17:23)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async AadhaarController.processOCR (C:\\Users\\phnaj\\aadhaar-ocr\\backend\\dist\\controllers\\aadhar.controller.js:45:39)","timestamp":"2025-10-24 17:26:05"}
{"level":"error","message":"OCR processing error: Aadhaar number already exists","service":"aadhaar-ocr","stack":"Error: Aadhaar number already exists\n    at AadhaarRepository.create (C:\\Users\\phnaj\\aadhaar-ocr\\backend\\dist\\repositories\\aadhar.repository.js:17:23)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async AadhaarController.processOCR (C:\\Users\\phnaj\\aadhaar-ocr\\backend\\dist\\controllers\\aadhar.controller.js:45:39)","timestamp":"2025-10-24 17:26:09"}
{"level":"error","message":"OCR processing error: Aadhaar number already exists","service":"aadhaar-ocr","stack":"Error: Aadhaar number already exists\n    at AadhaarRepository.create (C:\\Users\\phnaj\\aadhaar-ocr\\backend\\dist\\repositories\\aadhar.repository.js:17:23)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async AadhaarController.processOCR (C:\\Users\\phnaj\\aadhaar-ocr\\backend\\dist\\controllers\\aadhar.controller.js:45:39)","timestamp":"2025-10-24 17:26:23"}
{"level":"error","message":"OCR processing error: Aadhaar number already exists","service":"aadhaar-ocr","stack":"Error: Aadhaar number already exists\n    at AadhaarRepository.create (C:\\Users\\phnaj\\aadhaar-ocr\\backend\\dist\\repositories\\aadhar.repository.js:17:23)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async AadhaarController.processOCR (C:\\Users\\phnaj\\aadhaar-ocr\\backend\\dist\\controllers\\aadhar.controller.js:45:39)","timestamp":"2025-10-28 18:48:45"}
{"level":"error","message":"OCR processing error: Aadhaar number already exists","service":"aadhaar-ocr","stack":"Error: Aadhaar number already exists\n    at AadhaarRepository.create (C:\\Users\\phnaj\\aadhaar-ocr\\backend\\dist\\repositories\\aadhar.repository.js:17:23)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async AadhaarController.processOCR (C:\\Users\\phnaj\\aadhaar-ocr\\backend\\dist\\controllers\\aadhar.controller.js:45:39)","timestamp":"2025-10-28 23:40:14"}
{"level":"error","message":"OCR processing error: Aadhaar number already exists","service":"aadhaar-ocr","stack":"Error: Aadhaar number already exists\n    at AadhaarRepository.create (C:\\Users\\phnaj\\aadhaar-ocr\\backend\\dist\\repositories\\aadhar.repository.js:17:23)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async AadhaarController.processOCR (C:\\Users\\phnaj\\aadhaar-ocr\\backend\\dist\\controllers\\aadhar.controller.js:45:39)","timestamp":"2025-10-28 23:40:24"}
{"level":"error","message":"OCR processing error: Aadhaar number already exists","service":"aadhaar-ocr","stack":"Error: Aadhaar number already exists\n    at AadhaarRepository.create (C:\\Users\\phnaj\\aadhaar-ocr\\backend\\dist\\repositories\\aadhar.repository.js:17:23)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async AadhaarController.processOCR (C:\\Users\\phnaj\\aadhaar-ocr\\backend\\dist\\controllers\\aadhar.controller.js:45:39)","timestamp":"2025-10-28 23:41:08"}
